Test/COD50300.TXT

```diff
@@ -0,0 +1,153 @@
+OBJECT Codeunit 50300 Test_DotNet_Encoding
```

Comment:
The tests should be moved in Codeunit 146026

Reply:
Fixed with latest commits